### PR TITLE
ESLint 9 + config

### PR DESCRIPTION
**Pull request by mnkiefer**
- Requires release of `@sap/eslint-plugin-cds@3.0.0`

### CODE REVIEW

**Review by daogrady — APPROVED** [review]

**Review by bugwelle — APPROVED** [review]

File: eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ...cds.configs.recommended,
+  "plugins": {
+    "@sap/cds": cds
+  },
+  "languageOptions": {
+    globals: {
+      ...cds.globals
+    }
+  }
+}
```
Comment (bugwelle) [inline eslint.config.js:35]:
```suggestion
module.exports = [{
  "files": ["**/*.js"],
  ...js.configs.recommended,
  "languageOptions": {
    "globals": {
      ...globals.browser,
      ...globals.node,
      ...globals.jest,
      ...globals.mocha,
      "es2022": true
  },
  "rules": {
    ...js.configs.recommended.rules,
    "no-console": "off",
    "require-atomic-updates": "off",
    "require-await": "warn",
    "no-unused-vars": ["warn", { "argsIgnorePattern": "_" }]
  }
},
{ 
  "files": ['**/*.cds', '**/*.csn', '**/*.csv'],
  ...cds.configs.recommended,
  "plugins": {
    "@sap/cds": cds
  },
  "languageOptions": {
    globals: {
      ...cds.globals
    }
  }
}]
```

=> you forgot the braces. :)

**Review by bugwelle — CHANGES_REQUESTED** [review]

File: eslint.config.js (hunk 1 above)
Comment (mnkiefer) [reply eslint.config.js:35]: Thanks, Andre! 👍

**Review by bugwelle — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -28,6 +28,9 @@
     "chai-subset": "^1.6.0",
     "semver": "^7"
   },
+  "peerDependencies": {
```
Comment (chgeo) [inline package.json:31]: Why not a devDependency?  peerDependencies is for plugins, but cap-samples is an app.
Also, why not open it up to eslint 9, e.g. use `>=8` ?

**Comment by mnkiefer** [comment]
Now at https://github.com/SAP-samples/cloud-cap-samples/pull/711

**Pull request closed without merging**